src/beeai_agents/agent.py

```diff
@@ -1,26 +1,27 @@
 import os
-from collections.abc import AsyncGenerator
 
-
-from acp_sdk import Annotations, MessagePart, Metadata
-from acp_sdk.models import Message
-from acp_sdk.models.platform import PlatformUIAnnotation, PlatformUIType
-from acp_sdk.server import Context, RunYield, RunYieldResume, Server
+from a2a.types import (
+    AgentCapabilities,
+    Message,
+)
+from a2a.utils.message import get_message_text
+from beeai_sdk.server import Server
+from beeai_sdk.server.context import Context
+from beeai_sdk.a2a.types import AgentMessage
+from beeai_sdk.a2a.extensions import AgentDetail
 
 server = Server()
 
 @server.agent(
-    metadata=Metadata(
-        annotations=Annotations(
-            beeai_ui=PlatformUIAnnotation(ui_type=PlatformUIType.HANDSOFF)
-        )
+    detail=AgentDetail(ui_type="hands-off"),
+    capabilities=AgentCapabilities(
```

Comment:
capabilities are not necessary to specify, streaming=True is default

(neither is agent detail - default is chat)